scripts/validate-licenses.ts

```diff
@@ -73,6 +73,14 @@ const ignoredPackages = [
   'true-case-path@1.0.2', // Apache-2.0 but broken license in package.json
   'uws@9.14.0', // zlib license
 
+  // * Need to publish these with LICENSE files
+  // see https://github.com/bazelbuild/rules_nodejs/pull/696
```

Comment:
https://github.com/bazelbuild/rules_nodejs/pull/696 is now merged, can we get a release that removes this TODO?

Reply:
well, the npm version numbers for the bazel packages are identical to the upstream version numbers. If I publish `@bazel/bazel@0.24.2` it will be confusing since the latest is https://github.com/bazelbuild/bazel/releases/tag/0.24.1